lib/ast.js

```diff
@@ -345,7 +345,7 @@ var AST_Toplevel = DEFNODE("Toplevel", "globals", {
 var AST_Expansion = DEFNODE("Expansion", "expression", {
     $documentation: "An expandible argument, such as ...rest, a splat, such as [1,2,...all], or an expansion in a variable declaration, such as var [first, ...rest] = list",
     $propdoc: {
-        expression: "AST_Symbol the thing to be expanded"
+        expression: "[AST_Symbol] the thing to be expanded"
```

Comment:
So I guess you can do `...[0, 1]`?

If so, then it should read `AST_Node` instead - that the square bracket is just part of the documentation formatting instead of denoting an array.